src/utils/coordinates/WGS84CoordinateSystem.class.js

```diff
@@ -0,0 +1,77 @@
+import CoordinateSystemBounds from '@/utils/coordinates/CoordinateSystemBounds.class'
+import StandardCoordinateSystem, {
+    PIXEL_LENGTH_IN_KM_AT_ZOOM_ZERO_WITH_256PX_TILES,
+} from '@/utils/coordinates/StandardCoordinateSystem.class'
+import { round } from '@/utils/numberUtils'
+
+export default class WGS84CoordinateSystem extends StandardCoordinateSystem {
+    constructor() {
+        super(
+            4326,
+            'WGS 84 (lat/lon)',
+            '+proj=longlat +datum=WGS84 +no_defs +type=crs',
+            new CoordinateSystemBounds(
+                -180.0,
+                180.0,
+                -90.0,
+                90.0,
+                // center of LV95's extent transformed with epsg.io website
+                [8.239436, 46.832259]
```

Comment:
Might be better to use `proj4` to dynamically re-project the LV95 extent here ?

Reply:
Or even better make this configurable in the config.js, this way if someone want to use our soft and want to center on france for example he only needs to change its config.js file and not the implementation.

Reply:
I think this needs some kind of startup rework, as proj4 is not yet configured when we instantiate these coordinate systems.
Plus it would not be super robust, as if I start the app with `.../?sr=3857` without specifying a center, it would not reproject the default center automatically (so it needs some more thoughts)
But I like this idea!